apps/ui/src/types.ts

```diff
@@ -247,6 +247,7 @@ export type Vote = {
   };
   proposal: number | string;
   choice: number | number[] | Record<string, number>;
+  reason: string;
```

Comment:
Can we not add it to `Vote` type yet? It's still not there for SX APIs so it might be confusing if it's in the type.

Once we have it on the API and it's being queries we can add it back.